packages/config/config/devices/0x014a/tstat-tbz500.json

```diff
@@ -0,0 +1,712 @@
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 5,
+			"defaultValue": 0
+		},
+		{
+			"#": "13",
+			"label": "Heating Stage 2 On Threshold",
+			"description": "Sets the delta from setpoint that stage 2 heating starts.",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 2,
+			"maxValue": 7,
+			"defaultValue": 2
+		},
+		{
+			"#": "14",
+			"label": "Heating Stage 2 Off Threshold",
+			"description": "Sets the delta from setpoint that stage 2 heating stops. Stage 2 turns off at setpoint + Delta Stage 2.",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 6,
+			"defaultValue": 0
+		},
+		{
+			"#": "15",
+			"label": "Aux Heat On Threshold",
+			"description": "Sets the delta from setpoint that stage 3 heating starts",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 3,
+			"maxValue": 8,
+			"defaultValue": 3
+		},
+		{
+			"#": "16",
+			"label": "Aux Heat Off Threshold",
+			"description": "Sets the delta from setpoint that stage 3 heating stops. Stage 3 turns off at setpoint + Delta Stage 3.",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 7,
+			"defaultValue": 0
+		},
+		{
+			"#": "17",
+			"label": "Cooling Stage 1 On Threshold",
+			"description": "Sets the delta from setpoint that stage 1 cooling starts.",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 7,
+			"defaultValue": 1
+		},
+		{
+			"#": "18",
+			"label": "Cooling Stage 1 Off Threshold",
+			"description": "Sets the delta from setpoint that stage 1 cooling stops. Stage 1 turns off at setpoint - Delta Stage 1.",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 6,
+			"defaultValue": 0
+		},
+		{
+			"#": "19",
+			"label": "Cooling Stage 2 On Threshold",
+			"description": "Sets the delta from setpoint that stage 2 cooling starts.",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 2,
+			"maxValue": 8,
+			"defaultValue": 2
+		},
+		{
+			"#": "20",
+			"label": "Cooling Stage 1 Off Threshold",
+			"description": "Sets the delta from setpoint that stage 2 cooling stops. Stage 2 turns off at setpoint - Delta Stage 2.",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 7,
+			"defaultValue": 0
+		},
+		{
+			"#": "21",
+			"label": "Mechanical Status",
+			"valueSize": 2,
+			"minValue": 1,
+			"maxValue": 256,
+			"unsigned": true,
+			"readOnly": true,
+			"options": [
+				{
+					"label": "MECH_H1",
+					"value": 1
+				},
+				{
+					"label": "MECH_H2",
+					"value": 2
+				},
+				{
+					"label": "MECH_H3",
+					"value": 4
+				},
+				{
+					"label": "MECH_C1",
+					"value": 8
+				},
+				{
+					"label": "MECH_C2",
+					"value": 16
+				},
+				{
+					"label": "PHANTOM_F",
+					"value": 32
+				},
+				{
+					"label": "MECH_F",
+					"value": 64
+				},
+				{
+					"label": "MANUAL_F",
+					"value": 128
+				},
+				{
+					"label": "Reserved",
+					"value": 256
+				}
```

Comment:
remove

Reply:
I contemplated not including this parameter, but since it is readOnly, I thought that the unit _might_ actually return it and would appear as "unknown" in the UI if I didn't, right? That's why I left it in. Let me know if you still think it is unnecessary.

Reply:
From my experience, "reserved" usually means not used. Let's remove it, worst case it shows 256 instead of Reserved.

Reply:
Sounds good. Will remove once we settle the other issues.